code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -38,6 +38,8 @@
 	/// Mob that fired this ammunition (the pilot pressing the trigger)
 	var/mob/source_mob
 	var/combat_equipment = TRUE
+	var/ammo_ignore_caves = FALSE
+	/// Ignores Caves when lining up FMs
```

Comment:
the codedoc goes above the var, not below it

Reply:
moved the comment above it!